components/QuizDialog.tsx

```diff
@@ -53,7 +53,7 @@ export default function QuizDialog({
     if (!showAnswer) {
       setShowAnswer(true);
       if (
-        answer.toLowerCase().trim() === quizCards[currentIndex].answer.toLowerCase().trim()
+        answer === quizCards[currentIndex].answer
```

Comment:
Bug #3:

 answers are case sensitive
you have to answer in the exact case you entered in the card, otherwise it will not award you the question. both the user input and the card answer should be casefolded to alleviate this.